src/platforms/dotnet/migration.mdx

```diff
@@ -66,101 +64,94 @@ catch (Exception e)
 }
 ```
 
-#### Namespace Change
+## Namespace Change
 
 Classes under `Sentry.Protocol` were moved to the root namespace `Sentry`. All except `Context` classes like `App`, `Device`, etc.
 
-#### ASP.NET
-
-ASP.NET (**not Core**) users need to install an additional package:
+## ASP.NET
 
-`Sentry.AspNet`.
+ASP.NET (**not Core**) users need to install an additional package, `Sentry.AspNet`. Without this package, HTTP related information will not be added to events.
 
-Without this, HTTP related information will not be added to events.
+You can plug the package into the `Init` as follows:
 
-You can plug it in to the `Init` as follows:
 ```
 SentrySdk.Init(o =>
 {
     o.AddAspNet();
 });
 ```
 
-The motivation was to remove the reference to `System.Web` to decouple the core of the package with any specific namespaces of the .NET Framework, which improves the experience from other environments such as game engines like Unity and [Godot](https://gatomalo.dev/blog/2020/03/21/error-monitoring-godot-sentry/#android).
+The motivation was to remove the reference to `System.Web` and decouple the core of the package from any specific namespaces of the .NET Framework. This change improves the experience from other environments, such as game engines like Unity and [Godot](https://gatomalo.dev/blog/2020/03/21/error-monitoring-godot-sentry/#android).
 
-#### ASP.NET Core Environment Casing
+## ASP.NET Core
 
-To match Sentry's default environment naming convention, unless explicitly set otherwise, the Sentry environment will be reported lowercase. That means if `ASPNET_ENVIRONMENT` is equal to `Production`, the Sentry SDK will report it as `production`.
+### Environment Casing
 
-#### ASP.NET Core `IncludeRequestPayload`
+To match Sentry's default environment naming convention, unless explicitly set otherwise, the Sentry environment will be reported in lowercase. That means if `ASPNET_ENVIRONMENT` is equal to `Production`, the Sentry SDK will report it as `production`.
+
+### Removal of `IncludeRequestPayload`
 
 The option `IncludeRequestPayload` that was deprecated in the Sentry SDK 2.0 has been removed.
 The replacement is the option `MaxRequestBodySize`.
 
-#### `IHub` and `ISentryClient` Received New Methods
+### New Method for `IHub` and `ISentryClient`
```

Comment:
I believe it should be then:

```suggestion
### New Methods on `IHub` and `ISentryClient`
```

In other words: we added new methods to the interfaces IHub and ISentryClient